filebeat/harvester/reader/reader.go

```diff
@@ -0,0 +1,24 @@
+package reader
+
+import (
+	"time"
+
+	"github.com/elastic/beats/libbeat/common"
+)
+
+// Message represents a reader event with timestamp, content and actual number
+// of bytes read from input before decoding.
+type Message struct {
+	Ts      time.Time     // timestamp the line was read
+	Content []byte        // actual line read
+	Bytes   int           // total number of bytes read to generate the line
```

Comment:
don't use `line` in comments, but say content. Other file types might differ here for example.